coordinatorlib/src/main/scala/com/socrata/datacoordinator/common/soql/SoQLRowLogCodec.scala

```diff
@@ -118,6 +118,8 @@ object SoQLRowLogCodec extends SimpleRowLogCodec[SoQLValue] {
         target.writeStringNoTag(id)
       case SoQLNull =>
         target.writeRawByte(-1)
+      case SoQLInterval(_) =>
+        ???
```

Comment:
So this is an undefined placeholder . . .   I'm assuming we can never actually get here?

Reply:
Interval is 1/3 of a data type.  We use it to handle query function argument but it cannot get out to result set.

Reply:
And you can't use that as column data type.